### ISSUE TITLE
Zulip crashes pulpbar: possible issues with system tray & icon loading

### ISSUE THREAD

**Issue by Abastro** [Abastro/pulpmonad#1]
Gives unhelpful error message:
```
pulp-taskbar: Icon 'Zulip1_32-panel' not present in theme Yaru (0)
```

Need to identify
1. Why the error message is so limited (Is it IOException?)
2. How could the missing icon make the entire code crash

**Comment by Abastro** [comment]
Fixed some in c5a4c3f316aebe48483bac01e08b4a32c9eef140, need to see how it does when notification comes.

**Comment by Abastro** [comment]
Works as intended now.
